### ISSUE TITLE
32p is down but code not very forgiving...

### ISSUE THREAD

**Issue by barbequesauce** [mylar3/mylar3#65]
```
04-Feb-2020 13:24:40 - DEBUG :: mylar.__init__.68 : ThreadPoolExecutor-0_1 : [32P-AUTHENTICATION] [LOGIN SUCCESS] Now preparing for the use of 32P keyed authentication...
Job "RSS Feeds (trigger: interval[0:30:00], next run at: 2020-02-04 18:54:39 UTC)" raised an exception
Traceback (most recent call last):
  File "/opt/mylar.py3/mylar/auth32p.py", line 74, in __init__
    mylar.INKDROPS_32P = int(math.floor(float(lses.inkdrops['results'][0]['inkdrops'])))
TypeError: 'NoneType' object is not subscriptable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/mylar3/.local/lib/python3.8/site-packages/apscheduler/executors/base.py", line 125, in run_job
    retval = job.func(*job.args, **job.kwargs)
  File "/opt/mylar.py3/mylar/rsscheckit.py", line 72, in run
    feed32p = auth32p.info32p()
  File "/opt/mylar.py3/mylar/auth32p.py", line 76, in __init__
    mylar.INKDROPS_32P = lses.inkdrops['results'][0]['inkdrops']
TypeError: 'NoneType' object is not subscriptable
```

**Comment by barbequesauce** [comment]
Labelling as enhancement as it's a rare situation, to say the least

**Comment by evilhero** [comment]
Should be fixed with PR #68 (might want to confirm before closing)

**Comment by barbequesauce** [comment]
she looks beautiful!
